build.sbt

```diff
@@ -19,6 +19,11 @@ lazy val buildSettings = Seq(
     homepage := Option(url("http://47deg.github.io/github4s/")),
     organizationHomepage := Option(new URL("http://47deg.com")),
     scalaVersion := "2.11.8",
+    crossScalaVersions := Seq("2.10.6", scalaVersion.value),
```

Comment:
There are some incompatibilities with the `RosHTTP` library ([see this build](https://travis-ci.org/47deg/github4s/jobs/185127926)). I'm going to send a PR for adding support to Scala 2.12 in `RosHTTP` but in the meantime, I think we should go forward with this. What do you think?